externals/kyuubi-flink-sql-engine/bin/flink-sql-engine.sh

```diff
@@ -52,16 +52,14 @@ else
   logback_conf_file="unused.xml"
 fi
 
-FULL_CLASSPATH="$FLINK_SQL_ENGINE_JAR:$FLINK_SQL_CLIENT_JAR:$CC_CLASSPATH:$FLINK_HADOOP_CLASSPATH"
```

Comment:
This change is unnecessary, if you want to change the order of jars, just modify itself rather than inline it

Reply:
And also, I don't think the change of this script proves the flink sql engine can run without hadoop.

We should pick the Hadoop jars in production if user provides it, but we should not include `hadoop-client-*` jars in UT, right?